src/multibody/model-item.hpp

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright (c) 2021 INRIA
+//
+
+#ifndef __pinocchio_multibody_model_item_hpp__
+#define __pinocchio_multibody_model_item_hpp__
+
+#include "pinocchio/multibody/fwd.hpp"
+
+
+namespace pinocchio
+{
+  template<typename Derived>
+  struct ModelItem : NumericalBase<Derived>
+  {
+    EIGEN_MAKE_ALIGNED_OPERATOR_NEW
+
+    typedef typename traits<Derived>::Scalar Scalar;
+    enum { Options = traits<Derived>::Options };
+    typedef SE3Tpl<Scalar,Options> SE3;
+
+    /// \brief Name of the geometry object
```

Comment:
The documentation is not aligned with the content.

Reply:
I don't see any difference in the alignment. What is the issue that you see?

Reply:
Geometry is too specific

Reply:
Missed it. Should be fixed now